rush.json

```diff
@@ -427,6 +427,14 @@
       "shouldPublish": false
     },
 
+    {
+      "packageName": "@kadena/kadena-docs",
+      "projectFolder": "packages/apps/kadena-docs",
+      "reviewCategory": "prototypes",
+      "skipRushCheck": true,
```

Comment:
Why do we `skipRushCheck`?
* https://rushjs.io/pages/configs/rush_json/#:~:text=//%20%22skipRushCheck%22%3A%20false
* https://rushjs.io/pages/commands/rush_check/

It shouldn't be necessary. Maybe for apps it's ok to use though